src/app/pages/account/users/user-list/user-list.component.spec.ts

```diff
@@ -111,47 +111,25 @@ describe('UserListComponent', () => {
     store$ = spectator.inject(MockStore);
   });
 
-  it('should show table rows', async () => {
+  it('should show table rows', () => {
     store$.overrideSelector(selectPreferences, { hideBuiltinUsers: true } as Preferences);
     store$.overrideSelector(selectUsers, fakeUserDataSource);
     store$.refreshState();
 
-    const table = await loader.getHarness(IxTableHarness);
-    const cells = await table.getCells(true);
     const expectedRows = [
-      ['Username', 'UID', 'Builtin', 'Full Name', ''],
+      ['Username', 'UID', 'Builtin', 'Full Name'],
       ['root', '0', 'Yes', 'root', ''],
       ['test', '1004', 'No', 'test', ''],
     ];
 
-    expect(cells).toEqual(expectedRows);
-  });
-
-  it('should have empty message when loaded and datasource is empty', async () => {
-    store$.overrideSelector(selectUsers, []);
-    store$.refreshState();
-
     spectator.detectChanges();
-
-    const emptyRow = await loader.getHarness(IxEmptyRowHarness);
-    const emptyTitle = await emptyRow.getTitleText();
-    expect(emptyTitle).toBe('No records have been added yet');
-  });
-
-  it('should have error message when can not retrieve response', async () => {
-    store$.overrideSelector(selectUserState, {
-      error: 'Users could not be loaded',
-    } as UsersState);
-    store$.refreshState();
-    store$.select(selectUsers).subscribe((snapshots) => {
-      expect(snapshots).toEqual([]);
+    const cells = spectator.queryAll('tr').map((tr) => {
```

Comment:
As a separate task we should create harnesses to use in tests.

Reply:
https://ixsystems.atlassian.net/browse/NAS-122527